### DISCUSSION TITLE
webpack4子应用怎么能取消跨域限制

### DISCUSSION THREAD

**Discussion by Niefee** [jd-opensource/micro-app#637] [Q&A]
webpack4子应用怎么能取消跨域限制，基座请求子应用页面，报：
`Access to XMLHttpRequest at 'http://localhost:9527/#/home' from origin 'http://localhost.xxx.com:8888' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.`

`http://localhost.xxx.com:8888`是子应用，基座请求页面报错了，子应用已经加了配置：
```js

  devServer: {
    headers: {
      'Access-Control-Allow-Origin': '*',
    },
  },
  dev: {
    headers: {
      'Access-Control-Allow-Origin': '*',
    },
}
```

**Comment by bailicangdu** [comment]
还是没设置对吧，仔细检查一下

**Comment by soyeon-lq** [comment]
headers: {
      'Access-Control-Allow-Origin': '*',
      'Access-Control-Allow-Methods': '*',
      'Access-Control-Allow-Headers': '*'
    }
将这个配置信息添加上。就不会触发跨域
